Makefile.Common

```diff
@@ -4,7 +4,7 @@ SRC_ROOT := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -v -timeout 300s --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 300s
+GOTEST_INTEGRATION_OPT?= -race -timeout 360s
```

Comment:
I worry that increasing this would just encourage others to keep extending what their system does for timeouts.

Is there a way we can add in the ability for a virtual clock or something of that effect if we are testing time based functionality?

Reply:
I see what you're saying but I'm less worried about this than I am with CI stability. Please open an issue with a proposal if you have an idea how this could be implemented.